### PR TITLE
Signing owner selection box on Manage Packages page can cause table to overflow

### LINKED ISSUE #5909: Signing owner selection box on Manage Packages page can cause table to overflow

### PR DESCRIPTION
**Pull request by xavierdecoster**
Fixes #5909 

Explanation of fix: https://stackoverflow.com/questions/10672586/how-to-make-select-elements-shrink-to-max-width-percent-style-within-fieldset

Screenshot:
image

### CODE REVIEW

**Comment by joelverhagen** [comment]
Could we put the CSS in the page's CSS file rather than inline?

**Comment by xavierdecoster** [comment]
@joelverhagen yes we can! :)

File: src/Bootstrap/dist/css/bootstrap-theme.css [hunk 1]
```diff
@@ -1205,6 +1205,10 @@ img.reserved-indicator-icon {
 .page-manage-packages .inner-table {
   margin-bottom: 0;
 }
+.page-manage-packages .requiredSigner {
```
Comment (joelverhagen) [inline src/Bootstrap/dist/css/bootstrap-theme.css]: nit: `required-signer`

Comment (xavierdecoster) [reply src/Bootstrap/dist/css/bootstrap-theme.css]: Fixed!

**Pull request merged**